utils/rowgroup/rowgroup.h

```diff
@@ -343,14 +343,17 @@ class Row
     template<int len> inline bool equals(uint64_t val, uint32_t colIndex) const;
     inline bool equals(long double val, uint32_t colIndex) const;
     bool equals(const std::string& val, uint32_t colIndex) const;
-
+    inline bool equals(int128_t val, uint32_t colIndex) const;
```

Comment:
It could be. I don't think it matters very much. If you like that better, I'll change it. It might save one mv assembly code, depending on the micro code implementation.

Reply:
I wet ahead and made these const ref, since it will save a couple of pico seconds